curriculum/module_1/3_intro_to_pointers.md

```diff
@@ -14,7 +14,7 @@ A pointer is a variable which can store the address of a variable or value.
 
 *Examples*
 ```rust
-let x: u8 = 1234521; // requires 1 byte(8 bits) to store "x" in memory
```

Comment:
Could you keep the u8 type with a fitting value?

Reply:
I think I don't understand. If I bring this back to `u8`, then y and z should be `&u8` and `&&u8` right? 

Then do we also want to modify the struct too?